src/VueLazyYoutubeVideo.vue

```diff
@@ -129,6 +133,56 @@ export default Vue.extend({
         return ''
       }
     },
+    queryPart(): string {
```

Comment:
Why did you introduce query part? I think we can just leave `query` as computed property as is.

Reply:
autostarting a video for the background, for example, requires the video to be silent (before the user interacts with the site), if you pass the autoplay parameter to true but deny this in query then in this case the video will not be autostarted, so props autoplay is more priority, and this same parameter removes lazy loading, you need to work with it (as an option, start playback automatically if the video is in viewport)

Reply:
I’ll do a new pull request for lazy startup